NPB3.3-MPI/make.def

```diff
@@ -0,0 +1,162 @@
+#---------------------------------------------------------------------------
```

Comment:
Can you try to use the default make.def file with gfortran instead mpif77?
It can resolve all problems with problematic apps